convert_lora_to_gguf.py

```diff
@@ -363,7 +363,11 @@ def get_tensors(self) -> Iterator[tuple[str, Tensor]]:
                     yield (name, cast(torch.Tensor, LoraTorchTensor(tensor.A, tensor.B)))
 
             def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iterable[tuple[str, Tensor]]:
-                dest = super().modify_tensors(data_torch, name, bid)
+                dest = list(super().modify_tensors(data_torch, name, bid))
+                # for now, we cannot convert archs that use the same tensor for tok_embd and output
```

Comment:
```suggestion
                # for now, we cannot convert adapters with lm_head for archs that use the same tensor for tok_embd and output
```

Reply:
I’ll rephrase this to void mixing naming scheme